examples/purchaseTesterTypescript/app/screens/CustomerCenterModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import RevenueCatUI from 'react-native-purchases-ui';
+import { requireNativeComponent, View, StyleSheet, NativeSyntheticEvent } from 'react-native';
+
+const CustomerCenterView = requireNativeComponent('CustomerCenterView');
```

Comment:
we need to also create a view that wraps this native component, as we do in Paywalls (take a look at `Paywall` in index.tsx). At the moment this is just importing the native component directly from the `CustomerCenterViewManager`

We would also need to add support for an Android native view before we can merge this